redi-connect-front/src/pages/front/login/Login.tsx

```diff
@@ -70,144 +53,127 @@ export default function Login() {
     try {
       const accessToken = await login(formValues.username, formValues.password);
       saveAccessToken(accessToken);
-      const myProfile = await fetchSaveRedProfile(accessToken);
-      history.push("/app/dashboard");
+      await fetchSaveRedProfile(accessToken);
+      history.push('/app/dashboard');
     } catch (err) {
-      setLoginError("Invalid username or password");
+      setLoginError('You entered an incorrect email, password, or both.');
     }
     actions.setSubmitting(false);
   };
 
-  return (
-    <LoggedOutLayout>
-      <Formik
-        initialValues={initialValues}
-        validationSchema={validationSchema}
-        onSubmit={submitForm}
-        render={props => <Form {...props} loginError={loginError} />}
-      />
-    </LoggedOutLayout>
-  );
-}
+  const formik = useFormik({
+    // not sure how to solve the initial validation nicer...
+    // as the isInitialValid was depricated in the V2
+    initialErrors: { username: 'required' },
```

Comment:
I didn't check the `useFormik()` docs in detail yet, but on a whim removed this line (with `initialErrors`). The login form seemed to behave just like before. At first glance the idea of having an initial error with the form and all fields in a pristine state seems counter-intuitive?

Reply:
Oh... good catch, this is a merge of thinking gone wrong. Thanks for catching.